src/components/CheckEditor/CheckSettings.tsx

```diff
@@ -34,5 +34,8 @@ export const CheckSettings: FC<Props> = ({ isEditor, typeOfCheck }) => {
     case CheckType.K6: {
       throw new Error('Invalid check type for this location');
     }
+    case CheckType.GRPC: {
+      throw new Error('Invalid check type for this location');
+    }
```

Comment:
To be as accurate as possible I've decided not to shy away from the GRPC type checking but make sure it is valid and leave it in as it is something that will be worked on soon.